packages/core/src/shared/index.ts

```diff
@@ -70,3 +70,4 @@ export * from './lsp/utils/cleanup'
 export { default as request } from './request'
 export * from './lsp/utils/platform'
 export * as processUtils from './utilities/processUtils'
+export * from './lsp/lspInstaller'
```

Comment:
we need a lint rule for this :) 

Exporting all the symbols directly risks conflict with symbols from other modules. Also for consumers of the index.ts file, exporting an alias gives the consumer a "namespace" which is helpful.

```suggestion
export * as lspInstaller from './lsp/lspInstaller'
```